fv3core/testing/translate.py

```diff
@@ -295,18 +295,8 @@ def make_grid_storage(self, pygrid):
                 del self.data[k]
         for k, axis in TranslateGrid.edge_var_axis.items():
             if k in self.data:
-                edge_offset = pygrid.local_to_global_indices(pygrid.isd, pygrid.jsd)[
-                    axis
-                ]
-                if axis == 0:
-                    edge_offset = pygrid.global_isd
-                    width = pygrid.subtile_width_x
-                else:
-                    edge_offset = pygrid.global_jsd
-                    width = pygrid.subtile_width_y
-                edge_slice = slice(int(edge_offset), int(edge_offset + width + 1))
                 self.data[k] = utils.make_storage_data(
-                    self.data[k][edge_slice],
+                    self.data[k],  # [edge_slice],
```

Comment:
The 7.2.5 fortran data saved edge variables such as edge_s, edge_n, etc as they existed in the Fortran code. When they were used in the python code it was discovered that they were tile global variables, and this code was introduced to slice out the local bits. In 7.2.6 data, we have changed this (see https://github.com/ai2cm/fv3gfs-fortran/pull/223/files#diff-743dfc1cd3916f0ad907a0c35492bcf1eb8b86535d2932162b30135c344d96d5) so that the saved edge variables are the local compute slice needed. We do not need the commented out #[edge_slice], it is probably leftover from testing the change.

Reply:
Thanks for the response @rheacangeo